### PR TITLE
Enh/custom mocha report dir for screenshot

### PR DESCRIPTION
**Pull request by PeterNgTr**
**What are you trying to achieve?**
I changed the report directory when running codeceptjs with mochawesome report.

**What do you get instead?**
The failed screenshots get saved in the "output" default report directory and not the in the new directory and hence the report is unable to access is when served.

This fixed https://github.com/Codeception/CodeceptJS/issues/1371

### CODE REVIEW

File: lib/helper/Puppeteer.js [hunk 1]
```diff
@@ -1335,7 +1335,14 @@ class Puppeteer extends Helper {
    */
   async saveScreenshot(fileName, fullPage) {
     const fullPageOption = fullPage || this.options.fullPageScreenshots;
-    const outputFile = path.join(global.output_dir, fileName);
+    let outputFile;
+
+    if (!fileName.includes('/')) {
```
Comment (DavertMik) [inline lib/helper/Puppeteer.js]: copy-paste detected :)
let's move it somewhere to `utils`

Comment (PeterNgTr) [reply lib/helper/Puppeteer.js]: moved it to `utils`

**Comment by DavertMik** [comment]
:+1:

File: lib/utils.js [hunk 2]
```diff
@@ -268,3 +268,10 @@ module.exports.deleteDir = function (dir_path) {
     fs.rmdirSync(dir_path);
   }
 };
+
+module.exports.screenshotOutputFolder = function (fileName) {
+  if (!fileName.includes('/')) {
```
Comment (trinhpham) [inline lib/utils.js]: This causes problem on Windows.
Why didn't you use `path.sep`?

**Comment by trinhpham** [comment]
Found that screenshot won't be saved anymore on Windows because of this commit.
I've added comment to the bad code line.
@DavertMik + @PeterNgTr please help review again.

**Comment by PeterNgTr** [comment]
sure @trinhpham which line is it?

File: lib/utils.js (hunk 2 above)
Comment (PeterNgTr) [reply lib/utils.js]: I missed it. will fix

Comment (trinhpham) [reply lib/utils.js]: Great, @PeterNgTr !
When could we have a new build for this fix?
I've just upgraded CodeceptJS version today and going to downgrade it now 😢

Comment (PeterNgTr) [reply lib/utils.js]: https://github.com/Codeception/CodeceptJS/pull/1395
I fixed it here, can you try it on windows. I don't have windows machine :)

new build push should depend on @DavertMik

Comment (trinhpham) [reply lib/utils.js]: Glad to know version 2.0.1 has your fix.
Thanks 😄

**Pull request merged**